Tasks/DownloadPackage/download.ts

```diff
@@ -36,11 +37,18 @@ function getAuthToken() {
 	}
 }
 
-export async function downloadPackage(coreApi: corem.ICoreApi, feedId: string, packageId: string, version: string, downloadPath: string) {
-	var packageUrl = await getNuGetPackageUrl(coreApi.vsoClient, feedId, packageId);
-
+export async function downloadPackage(collectionUrl: string, credentialHandler: bearm.BearerCredentialHandler, feedId: string, packageId: string, version: string, downloadPath: string) {
+	
+	var feedsUrl = collectionUrl.replace(".visualstudio.com",".feeds.visualstudio.com");
```

Comment:
for dev. fabric this wouldn't work. since we don't have this working in dev. fabric as of now, it seems fine. but we need to figure out how to make this work in dev. fabric as part of adding L2 test for this scenario. let's take up this investigation then.